CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ cmake_minimum_required(VERSION 3.16)
 project(swift-atomics
   LANGUAGES C Swift)
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
```

Comment:
"The path to the source directory currently being processed."
https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_SOURCE_DIR.html